api.d.ts

```diff
@@ -186,6 +186,53 @@ export interface BundlingAPI {
 
 export declare const bundling: BundlingAPI
 
+// ----------------------------------------------------------------
+// -------------------------- SDK ----------------------------
+// ----------------------------------------------------------------
+
+export interface SdkAPI {
```

Comment:
I think least 2-3 people should review the APIs and confirm that those have been reviewed